src/AVFoundation/Enums.cs

```diff
@@ -211,7 +220,7 @@ public enum AVError : nint {
 		IncompatibleAsset = -11848,
 		FailedToLoadMediaData = -11849,
 		ServerIncorrectlyConfigured = -11850,
-		ApplicationIsNotAuthorizedToUseDevice = 11852,
+		ApplicationIsNotAuthorizedToUseDevice = -11852,
 			
```

Comment:
The above to sign fixes already have a PR on master to fix them asap: https://github.com/xamarin/xamarin-macios/pull/577